arch/xtensa/src/esp32/esp32_imm.c

```diff
@@ -73,6 +73,46 @@ FAR void *xtensa_imm_malloc(size_t size)
   return mm_malloc(&g_iheap, size);
 }
 
+/****************************************************************************
+ * Name: xtensa_imm_calloc
+ *
+ * Description:
+ *   Calculates the size of the allocation and
+ *   allocate memory the internal heap.
+ *
+ ****************************************************************************/
+
+FAR void *xtensa_imm_calloc(size_t n, size_t elem_size)
+{
+  return mm_calloc(&g_iheap, n, elem_size);
+}
+
+/****************************************************************************
+ * Name: xtensa_imm_realloc
+ *
+ * Description:
+ *   Reallocate memory from the internal heap.
+ *
+ ****************************************************************************/
+
+FAR void *xtensa_imm_realloc(void *ptr, size_t size)
```

Comment:
please change to: "FAR void *ptr"
Note that the use of FAR is not required inside arch/ and boards/ directories. It is used only on generic code that will be used by all architectures

Reply:
@acassis  I have modified it,  thank you.